set -e
true
true
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-hammer /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/corrected/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/python3 /Users/josephsada/miniconda3/envs/genomeanalysis/share/spades/spades_pipeline/scripts/compress_all.py --input_file /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/corrected/corrected.yaml --ext_python_modules_home /Users/josephsada/miniconda3/envs/genomeanalysis/share/spades --max_threads 16 --output_dir /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/corrected --gzip_output
true
true
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K21/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K33/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K55/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K77/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K99/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/spades-core /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/configs/config.info
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/python3 /Users/josephsada/miniconda3/envs/genomeanalysis/share/spades/spades_pipeline/scripts/copy_files.py /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/before_rr.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/before_rr.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/assembly_graph_after_simplification.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/assembly_graph_after_simplification.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/final_contigs.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/contigs.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/first_pe_contigs.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/first_pe_contigs.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/strain_graph.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/strain_graph.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/scaffolds.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/scaffolds.fasta /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/scaffolds.paths /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/scaffolds.paths /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/assembly_graph_with_scaffolds.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/assembly_graph_with_scaffolds.gfa /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/assembly_graph.fastg /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/assembly_graph.fastg /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/K127/final_contigs.paths /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/contigs.paths
true
/Users/josephsada/miniconda3/envs/genomeanalysis/bin/python3 /Users/josephsada/miniconda3/envs/genomeanalysis/share/spades/spades_pipeline/scripts/breaking_scaffolds_script.py --result_scaffolds_filename /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/scaffolds.fasta --misc_dir /Users/josephsada/Desktop/Bioinformatics/Final/spadesout/misc --threshold_for_breaking_scaffolds 3
true
